hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncTableImpl.java

```diff
@@ -44,12 +45,11 @@
 @InterfaceAudience.Private
 class AsyncTableImpl implements AsyncTable<ScanResultConsumer> {
 
-  private final AsyncTable<AdvancedScanResultConsumer> rawTable;
+  private final RawAsyncTableImpl rawTable;
```

Comment:
I think that changing this member variable to the concrete type makes following the relationship between these two classes more clear -- should I break this cleanup out into a separate PR?